Examples/MagicWeather/android/app/src/debug/AndroidManifest.xml

```diff
@@ -0,0 +1,7 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="m4trixdevlimited.com.flutter_magic_weather_app">
```

Comment:
should we update this package name? in other examples we have `com.revenuecat.sample` if I am not wrong